.drone.yml

```diff
@@ -0,0 +1,35 @@
+clone:
+  git:
+    image: plugins/git
+    depth: 50
+    tags: true
+
+cache: [".npm"]
+
+pipeline:
+  npm-install:
+    group: install
+    image: node:8-alpine
+    commands: ["npm install"]
+
+  eslint:
+    group: install
+    image: marcbachmann/eslint:4.19.1
```

Comment:
maybe start moving these images to livingdocs as we start using drone ci more regularly?